tigerpath/majors_and_certificates/majors/GER_2018.json

```
@@ -0,0 +1,403 @@
+          "req_list": [
+            {
+              "name": "Linguistics",
+              "max_counted": 1,
+              "min_needed": "ALL",
+              "explanation": "Majors who select this track are required to take Linguistics 213 (Introduction to Language and Linguistics) or a comparable course in linguistics.",
+              "course_list": [
+                "LIN 213",
+              ]
+            },
+            {
+              "name": "Germanic Linguistics",
+              "max_counted": 1,
+              "min_needed": "ALL",
+              "explanation": "Two seminars related to topics in Germanic linguistics: GER 505 (History of the German Language) or GER 508, and GER 506 (Second Language Acquisition and Pedagogy) or GER 316.",
+              "req_list": [
+                {
+                  "name": null,
+                  "max_counted": 1,
+                  "min_needed": null,
+                  "explanation": null,
+                  "course_list": [
+                    "GER 505",
+                    "GER 508"
+                  ]
+                },
+                {
+                  "name": null,
+                  "max_counted": 1,
+                  "min_needed": null,
+                  "explanation": null,
+                  "course_list": [
+                    "GER 506",
+                    "GER 316"
+                  ]
+                }
+              ]
+            },
+            {
+              "name": "German",
+              "max_counted": 1,
+              "min_needed": 3,
+              "explanation": "Students will take at least three other courses in German literature and culture (usually no more than two at the 200 level).",
+              "req_list": [
+                {
+                  "name": null,
+                  "max_counted": null,
+                  "min_needed": null,
+                  "explanation": null,
+                  "course_list": [
+                    "GER 3*",
+                    "GER 4*",
+                    "GER 5*"
+                  ]
+                },
+                {
+                  "name": null,
+                  "max_counted": 2,
+                  "min_needed": null,
+                  "explanation": null,
+                  "course_list": [
+                    "GER 2*"
+                  ]
+                }
+              ]
+            },
+            {
+              "name": "Cognates",
+              "max_counted": 1,
+              "min_needed": 2,
+              "explanation": "Two cognate courses.",

```

They are more explicit in the [handbook](https://german.princeton.edu/sites/default/files/2019-11/2019-2020%20current%20Undergraduate%20Handbook%20Nov.pdf).
```suggestion
              "explanation": "Two cognate courses pertaining to linguistics.",
```